src/patterns/data/default/entity_attribute_location.tsv

```diff
@@ -202,3 +202,5 @@ OBA:VT0008496	blood immunoglobulin G2a amount	PR:000050259	IgG2a heavy chain pro
 OBA:VT0003457	blood ketone body amount	CHEBI:73693	ketone body	PATO:0000070	amount	UBERON:0000178	blood
 OBA:VT0010476	blood glutamate dehydrogenase amount	GO:1990148	glutamate dehydrogenase complex	PATO:0000070	amount	UBERON:0000178	blood
 OBA:VT0010540	urine creatinine amount	CHEBI:16737	creatinine	PATO:0000070	amount	UBERON:0001088	urine
+OBA:VT0010441	parasite quantity	OPL:0000232	parasite organism	PATO:0000070	amount	UBERON:0000468	multicellular organism
+OBA:VT0010440	bacteria quantity	NCBITaxon:2	Bacteria	PATO:0000070	amount	UBERON:0000468	multicellular organism
```

Comment:
is this how we want to deal with species? Is the precedent anywhere in the file?

Reply:
No, I am afraid I could not find any precedent in the data files.
On second thought, the `entity_attribute_location` pattern is not the right one for these *host-[symbiont, parasite, infectious organism]* interactions from the point of view of the host (as a trait of the host), because the infecting organism should not be regarded a `part_of` the host.
Would the axiom below using [RO:0002453 host of](http://purl.obolibrary.org/obo/RO_0002453) work for these?
```
equivalentTo:
  text: "%s and 'characteristic_of' some (
    %s and
    ('host_of' some %s)
    )"
  vars:
    - attribute
    - host
    - parasite
```

Reply:
Can you ask on slack about a suitable pattern for this scenario?